<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FirstSetupWidget</class>
    <widget class="QWidget" name="FirstSetupWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>466</width>
                <height>228</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <widget class="QPushButton" name="create_button">
                    <property name="text">
                        <string>Erste Regelgruppe erstellen</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <widget class="QPushButton" name="import_local_button">
                            <property name="text">
                                <string>Aus Datei importieren</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="import_online_button">
                            <property name="text">
                                <string>Aus dem Internet importieren</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
